R/expr-expr.R

```diff
@@ -1261,7 +1261,10 @@ expr__dot <- function(other) {
 #' df$select(pl$col("foo")$reshape(c(3, -1)))
 expr__reshape <- function(dimensions) {
   wrap({
-    if (is.numeric(dimensions) && anyNA(dimensions)) {
+    if (!is_integerish(dimensions, n = 2)) {
+      abort("`dimensions` only accepts up to 2 integer-ish values.")
+    }
```

Comment:
Sorry but I think I misunderstood, it is past time that reshape did not accept lengths longer than 2, doesn't it work now?

Reply:
Oh indeed, I got mistaken because our docs were outdated (actually we have a test with more than 2 dims). I'll fix that.